packages/webapp/src/util/siteMapConstants.ts

```diff
@@ -17,10 +17,13 @@
 //   as well as accomodating UUID types.
 
 // Finances
+
+export const FINANCES_URL = '/finances';
 export const FINANCES_HOME_URL = '/finances/transactions';
 export const REVENUE_TYPES_URL = '/finances/revenue_types';
 export const ADD_REVENUE_URL = '/finances/add_revenue';
 export const MANAGE_CUSTOM_REVENUES_URL = '/finances/manage_custom_revenues';
+export const ACTUAL_REVENUE_URL = '/finances/actual_revenue';
```

Comment:
Ohhhh! It looks like what might have happened here was that `actual_revenue` was never in the deprecated routes file (maybe it was always nested under finances to start with?)

Reply:
Done! Yeah, it seems it was already nested within finances before I made the changes

Reply:
Yes it was never deprecated so we left it Joyce when we tag teamed the file..  you mentioned you specifically didn't want it in the last patch... just in case haha.

Reply:
@Duncan-Brain ohhhhh! No it was the other two things you listed (renaming `sale` to `revenue` and `categories` to `types`) that I didn't want to mess with before the patch, and then I didn't understand what you meant with this one. But yeah seeing it now, it doesn't really matter whether it went to the hotfix or not.